client/src/containers/App.tsx

```diff
@@ -42,13 +48,169 @@ export default class App extends Component<Props, State> {
 
     this.state = {
       showEngageModal: false,
+      entryIndex: 0,
+      entries: [
+        {
+          index: "1 of 4",
+          title: "Timeline Tab",
+          description: "Click the Timeline tab to view info about your building over time.",
+          img: Timeline_gif,
+        },
+        {
+          index: "2 of 4",
+          title: "Spanish Support",
+          description:
+            "Who Owns What is now available in Spanish. Click “ES” in the upper right corner to switch your language.",
+          img: Spanish_gif,
+        },
+        {
+          index: "3 of 4",
+          title: "Unique Tab URLs",
+          description:
+            "It's now possible to share links to specific tabs (Overview, Timeline, Portfolio, & Summary).",
+          img: URLS_gif,
+        },
+        {
+          index: "4 of 4",
+          title: "Last Sold",
+          description: "The Overview and Portfolio tabs now display the date and price from when your building was last sold.",
+          img: LastSold_gif,
+        },
```

Comment:
So I assume that ideally this information should be pulled from Contentful, right?  Maybe with the exception of `index`, which we can generate on-the-fly?